### PR TITLE
Add velorent as an option for bike rental

### LINKED ISSUE #545: JuliaCon 2024: add bike rentals to website
TIP: RENT A BIKE and cycle from your hotel to the venue and the rest of the surroundings, starting from just € 9,50 per day! Grab your bike [here](mailto:info@velorent.nl) and let the adventure begin!

Can we add this to the website?

### PR DESCRIPTION
**Pull request by bvdmitri**
This PR fixes #545

Would that be ok @cityjumper ? We've already had some options, e.g. NS bike rental. I kept them on the website, but I put the velorent as a first option

### CODE REVIEW

**Comment by giordano** [comment]
Just to reiterate what I said in the other PR, you find the link to the preview in the CI checks, after the build finishes. For the record, it's here: https://juliacon.org/previews/PR547/

**Comment by bvdmitri** [comment]
Thanks @giordano , that is very helpful!

**Review by lucaferranti — APPROVED** [review]
Looks good. Do you want to make these links opening in a new tab too?

File: 2024/travel.md [hunk 1]
```diff
@@ -63,7 +63,7 @@ Eindhoven can also be easily reached by public transport from cities around the
 
 ### Accommodation options
 
-Eindhoven, a contemporary city with diverse accommodations in close proximity to the venue, serves as an ideal setting for our conference. JuliaCon organizers have secured exclusive promotional deals with select hotels in the area, providing attendees with a substantial **15% discount**. Detailed information about these special offers can be accessed [here](/2024/venue-hotels). Additionally, a range of alternative accommodations can be explored through reputable platforms such as [Booking.com](https://www.booking.com) or [AirBnb](https://www.airbnb.com). 
+Eindhoven, a contemporary city with diverse accommodations in close proximity to the venue, serves as an ideal setting for our conference. JuliaCon organizers have secured exclusive promotional deals with select hotels in the area, providing attendees with a substantial **15% discount**. Detailed information about these special offers can be accessed [here](/2024/venue-hotels). Additionally, a range of alternative accommodations can be explored through reputable platforms such as [AirBnb](https://www.airbnb.com).
```
Comment (bvdmitri) [inline 2024/travel.md:66]: See https://github.com/JuliaCon/www.juliacon.org/issues/528

**Comment by bvdmitri** [comment]
I would prefer to keep standard `.md` links whenever possible. Just to keep things simple. The hotels are exceptions, only because it was requested explicitly.

**Pull request merged**